### ISSUE TITLE
Make it clearer where the issues tracker is

### ISSUE THREAD

**Issue by fatso83** [janinko/ghprb#381]
Right now there are 3 different places to file bugs/issues:
- [janinko/ghprb](https://github.com/janinko/ghprb/issues) (this repo is behind the jenkins one, so perhaps close this down/mark it as obsolete?)
- [jenkinsci/ghprb]((https://github.com/jenkinsci/ghprb/issues)
- [JenkinsCi.org](https://issues.jenkins-ci.org/browse/JENKINS-32389?jql=project%20%3D%20JENKINS%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20component%20%3D%20'ghprb-plugin')

That must be both confusing for you reviewing the issues and the users of the plugin. Please close down unofficial issue trackers and improve the documentation for where to direct issues.

I have no idea where the best place to file any issues is.

**Comment by DavidTanner** [comment]
I completely agree with you.  I haven't seen any input from @janinko for several years, and an owner the r would be the only one that can remove the PR section here.  Also you can't remove the issues section from a repo in GitHub.  

The issues tracker in jira is created for every plugin for jenkins.  

I will see about adding some documentation to direct everyone to the jenkins fork for issues and pull requests.
